.travis/linux..install.sh

```diff
@@ -15,3 +15,11 @@ else
 fi
 
 sudo apt-get install -y $PACKAGES
+
+# Carla depends on kxstudio which creates some package conflicts (wine, etc)
+# If run too early in the dependency process.  Once provided by PPA, "carla-git"
+# can simply be added to $PACKAGES
+sudo apt-get install -y apt-transport-https software-properties-common wget
+wget https://launchpad.net/~kxstudio-debian/+archive/kxstudio/+files/kxstudio-repos_9.4.6~kxstudio1_all.deb
+sudo dpkg -i kxstudio-repos_9.4.6~kxstudio1_all.deb
```

Comment:
`sudo apt-get update` is missing here

Reply:
Also these steps will break as soon as I update that deb file.

Try this instead:
```
sudo add-apt-repository -y kxstudio-debian/gcc5
sudo add-apt-repository -y kxstudio-debian/gcc5-deps
sudo apt-get update
```

Reply:
@falkTX Thanks, but this isn't working on my machine.  how should I go about adding `kxstudio-debian/gcc5[...]` on Ubuntu 14.04?

Reply:
ah no, those are for 16.04.
I thought the AppImages was using that.

So use this:
```
sudo add-apt-repository -y kxstudio-debian/libs
sudo add-apt-repository -y kxstudio-debian/apps
sudo apt-get update
```

Reply:
Nevermind, I think it was just missing `ppa:` before the repository name.

Reply:
Ok, `carla-git` isn't showing up for `ppa:kxstudio-debian/gcc5`.  Did you mean to say `ppa:kxstudio-debian/kxstudio` ?

Reply:
Got your messages late.  Trying that now, thanks.